mobile/src/main/res/xml/preferences.xml

```diff
@@ -6,61 +6,81 @@
-            android:title="@string/settings_openhab_sslhost" />
+            android:title="@string/settings_openhab_demomode"/>
+        <PreferenceScreen android:title="@string/settings_openhab_connection">
+            <org.openhab.habdroid.ui.widget.URLInputPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_url"
+                android:summary="@string/settings_openhab_url_summary"
+                android:title="@string/settings_openhab_url"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_username"
+                android:summary="@string/settings_openhab_username_summary"
+                android:title="@string/settings_openhab_username"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textPassword"
+                android:key="default_openhab_password"
+                android:summary="@string/settings_openhab_password_summary"
+                android:title="@string/settings_openhab_password"/>
+        </PreferenceScreen>
+        <PreferenceScreen
+            android:title="@string/settings_openhab_alt_connection">
+            <org.openhab.habdroid.ui.widget.URLInputPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_alturl"
+                android:summary="@string/settings_openhab_alturl_summary"
+                android:title="@string/settings_openhab_alturl"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_remote_username"
+                android:summary="@string/settings_openhab_username_summary"
+                android:title="@string/settings_openhab_username"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textPassword"
+                android:key="default_openhab_remote_password"
+                android:summary="@string/settings_openhab_password_summary"
+                android:title="@string/settings_openhab_password"/>
+        </PreferenceScreen>
+        <PreferenceScreen android:title="@string/settings_openhab_sslsettings">
```

Comment:
Why convert SSL settings to a pref screen? No potential confusion here AFAICT?

Reply:
I don't think so. The SSL settings are common for both, the local and the remote settings, so they at least do not belong to one of them. We probably can create a separate PreferenceCategory for it, would that be ok?

Reply:
Yeah, using a PreferenceCategory sounds fine.